buddyboss/buddyboss-platform
# PROD-716 Fix slug accent issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3074 

### How to test the changes in this Pull Request:

1. Go to BuddyBoss setting page -> forum
2. Change the forum slug to `fórum`
3. Now go to frontend and check the forum creation is working while creating the group

### Other information:
